templates/drupal-11/.github/workflows/build_release.yml

```
@@ -0,0 +1,48 @@
+on:
+  push:
+    tags:
+      - "*.*.*"
+
+name: Create Github Release
+
+permissions:
+  contents: write
+
+env:
+  COMPOSE_USER: runner
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    env:
+      COMPOSER_ALLOW_SUPERUSER: 1
+      APP_ENV: prod
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Composer install
+        run: |
+          docker network create frontend
+          docker compose run --rm --user=root phpfpm composer install --no-dev -o --classmap-authoritative
```

When `env.COMPOSE_USER` is set to `runner`, we don't need `--user=…` here:

```suggestion
          docker compose run --rm phpfpm composer install --no-dev -o --classmap-authoritative
```